cloudformation/media-atom-maker.json

```diff
@@ -53,6 +53,23 @@
             "Type": "String"
         }
     },
+    "Mappings": {
+        "StageMap": {
+            "PROD": {
+                "MinSize": 1,
```

Comment:
There is currently just 1 in production so I kept it the same. I assume we'll be bumping it up in the near future but perhaps we're not at that point?